server/scan/device_handling.py

```
@@ -1,764 +1,816 @@
+- Python standard libraries: sys, subprocess, os, re
+- External modules: conf, helper, logger, const, models.device_instance, scan.name_resolution
+- Temporary libraries: warnings(removable when all deprecations are able to be fully removed)
+"""
+
 import sys
+import subprocess
+import os
+import re
+import warnings
+from typing import Optional, List, Tuple, Dict
+from datetime import datetime
```

Ruff (0.11.9)

`datetime.datetime` imported but unused

Remove unused import: `datetime.datetime`

(F401)